.github/workflows/edge.yml

```diff
@@ -23,6 +23,8 @@ on:
       - opened
       - reopened
       - synchronize
+  schedule:
+    - cron:  '0 4 * * *'
```

Comment:
shouldn't we run edge only on repo actions as usual

Reply:
This is because of putting nightly and edge in same file. The same pipeline steps will run in edge and nightly as they do now though, the only difference is how they are declared.